src/styles/blocks/_calendar.scss

```
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  padding: 10px;
+  width: calendar-width($gap, $size, $columns);
+  @each $day, $num  in $days {
+    &--start-day-#{$day} {
+      .calendar__item:first-child {
+        margin-left: start-day-margin($num);
+      }
+    }
+  }
+  &__item {
+    list-style: none;
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $size;
+    height: $size;
+    background-color: #eee;
+    border: 1px solid #000;
+    transition: all 0.5s;
+    &:hover {
```

Don't use all for less than 4 rules to transition.
```suggestion
    transition: all 0.5s;
    
    &:hover {
```